indico/modules/rb/blueprint.py

```diff
@@ -32,6 +32,9 @@
 _bp.add_url_rule('/api/stats', 'stats', misc.RHStats)
 _bp.add_url_rule('/api/map-areas', 'map_areas', misc.RHMapAreas)
 _bp.add_url_rule('/api/equipment', 'equipment_types', misc.RHEquipmentTypes)
+_bp.add_url_rule('/api/tos', 'tos', misc.RHTos)
+_bp.add_url_rule('/api/help', 'help_url', misc.RHHelp)
```

Comment:
Do we really need API endpoints for this? I think this can all go into `RHConfig` - I'd go as far as including the ToS URL/text in there, so none of the new endpoints here would be needed.